docs/user-guide/deployment/kserve.md

```diff
@@ -0,0 +1,149 @@
+
+```{note}
+This section assumes a basic knowledge of KServe and Kubernetes, as well as
+access to a working Kubernetes cluster with KServe installed.
+To learn more about [KServe](https://kserve.github.io/website/) or [how to
+install it](https://kserve.github.io/website/get_started/), please visit the
+[KServe documentation](https://kserve.github.io/website/).
+```
+
+## Serving Runtimes
+
+KServe provides built-in [serving
+runtimes](https://kserve.github.io/website/modelserving/v1beta1/serving_runtime/)
+to deploy models trained in common ML frameworks.
+These allow you to deploy your models into a robust infrastructure by just
+pointing to where the model artifacts are stored remotely.
+
+Some of these runtimes leverage MLServer as the core inference server.
+Therefore, it should be straightforward to move from your local testing to your
+serving infrastructure.
+
+### Usage
+
+To use any of the built-in serving runtimes offered by KServe, it should be
+enough to select the relevant one your `InferenceService` manifest.
+
+For example, to serve a Scikit-Learn model, you could use a manifest like the
+one below:
+
+```{code-block} yaml
+---
+emphasize-lines: 7, 8
+---
+apiVersion: serving.kserve.io/v1beta1
+kind: InferenceService
+metadata:
+  name: my-model
+spec:
+  predictor:
+    sklearn:
+      protocolVersion: v2
+      storageUri: gs://seldon-models/sklearn/iris
+```
+
+As you can see highlighted above, the `InferenceService` manifest will only
+need to specify the following points:
+
+- The model artifact is a Scikit-Learn model. Therefore, we will use the
+  `sklearn` serving runtime to deploy it.
+- The model will be served using the [V2 inference
+  protocol](https://kserve.github.io/website/modelserving/inference_api/),
+  which can be enabled by setting the `protocolVersion` field to `v2`.
+
+Once you have your `InferenceService` manifest ready, then the next step is to
+apply it to your cluster.
+There are multiple ways to do this, but the simplest is probably to just apply
+it directly through `kubectl`, by running:
+
+```bash
+kubectl apply -f my-inferenceservice-manifest.yaml
+```
+
+### Supported Serving Runtimes
+
+As mentioned above, KServe offers support for built-in serving runtimes, some
+of which leverage MLServer as the inference server.
+Below you can find a table listing these runtimes, and the MLServer inference
+runtime that they correspond to.
+
+| Framework    | MLServer Runtime                           | KServe Serving Runtime | Documentation                                                                                |
+| ------------ | ------------------------------------------ | ---------------------- | -------------------------------------------------------------------------------------------- |
+| Scikit-Learn | [MLServer SKLearn](../../runtimes/sklearn) | `sklearn`              | [SKLearn Serving Runtime](https://kserve.github.io/website/modelserving/v1beta1/sklearn/v2/) |
```

Comment:
kserve doesnt support mlflow?

Reply:
There isn't a top-level serving runtime exposing the MLServer one yet.